submissions/morenko/html-css-popup/style.css

```diff
@@ -0,0 +1,214 @@
+/* импортируем гугл шрифт */
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;600;800&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+}
+
+/* заведем переменные в руте*/
+:root {
+    /* Typography */
+    --family: 'Nunito Sans', sans-serif;
+  /* размер */
+    --fs-sm: 14px;
+    --fs-md: 16px;
+  /* насыщенность */
+    --fw-light: 300;
+    --fw-normal: 600;
+    --fw-bold: 800;
+  
+    /* бордер */
+    --radii: 0.5rem;
+    --border: 1px solid black;
+
+    --shadow: rgba(79, 79, 79, 1) 0 0 8px;
+
+    --colors-bg: hsla(203, 100%, 39%, 0.2);
+    --colors-bg-active: hsla(189, 16%, 91%, 0.686);
+}
+
+body {
+    /* background: url(../assets/img/1637352650_71-krot-info-p-goluboe-more-peizazh-krasivoe-foto-80.jpg) no-repeat center fixed;
+    background-size: cover; */
+    /* используем переменные ранее созданые */
+    font-family: var(--family);
+    /* основной цвет */
+    color: var(--colors-text);
+    font-weight: var(--fw-bold);
+    background-color: var(--colors-bg);
+}  
+li {
+    list-style-type: none;
+    text-decoration: none;
+}
+ul {
+    padding: 0;
+}
+
+.nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--colors-bg);
+}
+
```

Comment:
Try to keep your code style cleaner: space between code blocks, no unused lines inside code block(in CSS)